src/ACadSharp/IO/DWG/DwgStreamReaders/DwgObjectReader.cs

```diff
@@ -6641,6 +6648,91 @@ private void readPlotSettings(PlotSettings plot)
 				//Visual Style handle(soft pointer)
 				this.handleReference();
 		}
+        
+        private void readLinetypeSegmentTexts(IList<CadLineTypeTemplate.SegmentTemplate> segments, byte[] textArea)
+        {
+            if (segments == null || textArea == null || textArea.Length == 0)
+                return;
+
+            Encoding encoding = this._reader?.Encoding ?? Encoding.ASCII;
```

Comment:
In the writer I see that we use Unicode or ASCII, not the assigned reader encoding, has it been tested?